### DISCUSSION TITLE
What should we start including - What are the To Dos??

### DISCUSSION THREAD

**Discussion by nabilashbat** [ByteRunners/NextJS-Sprout#3] [Ideas]
Ideas and thoughts here:

**Comment by AhmadHRai** [comment]
Let's have a meeting to discuss the first sprint and what libraries to include in this template today

**Comment by nabilashbat** [comment]
Ready when you are 👍

Useful ink: [Awesome Next.js](https://github.com/officialrajdeepsingh/awesome-nextjs)

**Comment by nabilashbat** [comment]
List of suggested tech stack to be implemented:

[Next.js Template Tech Stack](https://gist.github.com/nabily4e-dev/3a97af4394c5878919e9f830475fe428#file-nexjs-template-tech-stack-md)
